.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
+* * *
+
+**JIRA Ticket**: (link)
```

Comment:
Suggestion: change to:
```
**Resolves Issue**: (GitHub Issue)

Reply:
Fixed. BTW, this PR template was copied from another project so likely all other projects have `**JIRA Ticket**`